From 8750db6f679af7339def419b4fcf9683abd33910 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Wed, 12 Nov 2014 16:00:54 +0530
Subject: [PATCH 1/1] PMIC-BCU: Configure VWARNB value greater than VWARNA

As the re-active throttling components are connected in BCUDISB pin, the VWARNB
threshold value should be configured greater than the value of VWARNA. So that
the re-active components throttling will happen first for VWARNB event and then
software graceful shutdown for VWARNA event.

Tracked-on: https://jira01.devtools.intel.com/browse/IMINAN-8810
Change-Id: Iff2c3a81d893c65ae0d3d0ac875ae90e532200f2
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
---
 drivers/mfd/intel_soc_pmic_wc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mfd/intel_soc_pmic_wc.c b/drivers/mfd/intel_soc_pmic_wc.c
index 157c9e8..e646cfa 100644
--- a/drivers/mfd/intel_soc_pmic_wc.c
+++ b/drivers/mfd/intel_soc_pmic_wc.c
@@ -262,7 +262,7 @@ static struct pmic_ccsm_int_cfg wc_intmap[] = {
 static struct wcove_bcu_platform_data wc_bcu_pdata = {
 	.config = {
 		{VWARNA_CFG_REG,	0xFF},
-		{VWARNB_CFG_REG,	0xFF},
+		{VWARNB_CFG_REG,	0xFC},
 		{VCRIT_CFG_REG,		0xFD},
 		{ICCMAXVCC_CFG_REG,	0x06},
 		{ICCMAXVNN_CFG_REG,	0x06},
-- 
1.9.1

